### ISSUE TITLE
`request includes an unrecognized parameter 'value'` should be `parameter 'username'`

### ISSUE THREAD

**Issue by cunnie** [cloudfoundry-incubator/credhub-cli#37]
What version of the credhub server you are using?

Don't know; I assume the error message is coming from the CLI; if it's from the server, I need to explore this more deeply.

What version of the credhub cli you are using?

1.6.0

If you were attempting to accomplish a task, what was it you were attempting to do?

```
./credhub generate   --name=$PREFIX_PROD/rabbitmq-odb-ch-prod   --type=password   --username=$USER   --no-overwrite
```

What did you expect to happen?

I expected to get an error similar to `request includes an unrecognized parameter 'username'`

What was the actual behavior?

`The request includes an unrecognized parameter 'value'`

Please confirm where necessary:
* [ ] I have included a log output
* [ ] My log includes an error message
* [x] I have included steps for reproduction

If you are a PCF customer with an Operation Manager (PCF Ops Manager) please direct your questions to support (https://support.pivotal.io/)

**Comment by crawsible** [comment]
@cunnie Thanks for reporting; looks like a simple error on our part. Scheduling.

**Comment by crawsible** [comment]
@cunnie This fix is present in 1.7.3. Can you give it a whirl and close the issue?

**Comment by cunnie** [comment]
Thanks for fixing @crawsible, sorry I missed your request-to-test (I'm shamefully behind on GitHub notifications)
